### ISSUE TITLE
a problem

### ISSUE THREAD

**Issue by john-yyb** [kalpeshjp89/USISResNet#9]
Hello author, I am very interested in your work. I tried to reproduce it but ran into a problem, can you tell me how you solved it.
/home/student/student9/miniconda3/lib/python3.10/site-packages/torch/optim/lr_scheduler.py:131: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
/home/student/student9/miniconda3/lib/python3.10/site-packages/torch/optim/lr_scheduler.py:418: UserWarning: To get the last learning rate computed by the scheduler, please use `get_last_lr()`.

**Comment by kalpeshjp89** [comment]
Hi,
The message you posted here is just a warning. You can ignore it.

Regards,
Kalpesh

**Comment by john-yyb** [comment]
Thank you very much for your prompt response. For your work, can I try swapping the dataset. I tried using a dataset of black and white images. For this data set, I intend to downsample one part as LR and the other part as HR. Then the two form an LR-HR unpaired dataset for training. Am I doing the right thing? Because I am new to super-resolution of unsupervised learning, there may be incorrectly understood. I hope I can get your answer, then I will be very grateful.

**Comment by kalpeshjp89** [comment]
Hi
Actually in unsupervised SR task, you have unpaired LR-HR dataset.
For example, consider images captured by low-end mobile device as LR and
download some high quality images from internet. With this type of data, if
you apply SR, then it is called unsupervised SR.
But if you created LR from some HR data, in that case you have paired LR-HR
data. In this case, you can use any supervised method for better results.

I hope, you can understand it now.

**Comment by john-yyb** [comment]
Thank you very much for your reply, I understand now.
